src/main/kotlin/net/idlestate/gradle/caching/GCSBuildCacheService.kt

```diff
@@ -104,4 +106,11 @@ class GCSBuildCacheService(credentials: String, val bucketName: String, val refr
     override fun close() {
         // nothing to do
     }
+
+    companion object {
+        /**
+         * The threshold to write data onto storage.
+         */
+        const val FILE_THRESHOLD = 8 * 1024 * 1024
```

Comment:
Honestry speaking, there is no strong reason to pick `8MiB` as threshold.

Reply:
I will probably add a configuration option for the threshold.